src/test/groovy/com/palantir/gradle/javadist/JavaDistributionPluginTests.groovy

```diff
@@ -62,10 +62,13 @@ class JavaDistributionPluginTests extends Specification {
 
         // try all of the service commands
         exec('dist/service-name-0.1/service/bin/init.sh', 'start') ==~ /(?m)Running 'service-name'\.\.\.\s+Started \(\d+\)\n/
+        sleep 5000
```

Comment:
Yes: my first commit without this failed and looking at the Travis builds it looks like PR#20 went through the same thing.